content/learning-paths/11111111-1111-1111-1111-111111111111/mastering-kubernetes-for-engineers/ambassador-edge-stack-api-gateway-with-meshery/conclusion/quiz.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Quiz"
+passing_percentage: 70
+layout: "test"
+type: "test"
+questions:
+  - id: "q1"
+    text: "What tool was used in this tutorial to visualize resource relationships in the Ambassador Edge Stack system?"
+    type: "single-answer"
+    marks: 2
+    options:
+      - id: "a"
+        text: "Kubectl"
+      - id: "b"
+        text: "Kanvas"
+        is_correct: true
+      - id: "c"
+        text: "Helm"
+  - id: "q2"
+    text: "What key learnings were covered in this Ambassador Edge Stack course? (Select all that apply)"
+    type: "multiple-answers"
+    marks: 2
+    options:
+      - id: "a"
+        text: "Exploring AES system using Meshery Playground"
+        is_correct: true
+      - id: "b"
+        text: "Configuring, deploying, and managing Ambassador Edge Stack with Meshery"
+        is_correct: true
+      - id: "c"
+        text: "Setting up database connections"
+  - id: "q3"
+    text: "What platform was used throughout this tutorial to work with Ambassador Edge Stack?"
+    type: "short_answer" 
+    marks: 2
+    correct_answer: "Meshery Playground" 
```

Comment:
@zihanKuang @Vincamine @aabidsofi19 what is the system behavior for "short_answers"? Do learners need to make the text exactly or just have the same text somewhere in the answer? Is validation of the answer case insensitive?